google-cloud-spanner/lib/google/cloud/spanner/database.rb

```diff
@@ -238,6 +238,9 @@ def ddl force: nil
         #   valid identifier: `[a-z][a-z0-9_]*`. Will raise
         #   {Google::Cloud::AlreadyExistsError} if the named operation already
         #   exists. Optional.
+        # @param [Google::Protobuf::FileDescriptorSet, String] descriptor_set The file
+        #   descriptor set object to be used in the update, or alternatively, an absolute
+        #   path to the generated file descriptor set.
```

Comment:
Mention this is applicable when using `CREATE PROTO BUNDLE` so users aren't confused as to what it's for.